lib/get-api.js

```diff
@@ -3,8 +3,8 @@ const { trim } = require('omnibelt');
 
 process.env.LOSANT_API_URL = process.env.LOSANT_API_URL || 'https://api.losant.com';
```

Comment:
I don't think there is any reason to *set* the env var any more, especially cause it might not be what is actually getting used some times. I'd probably just do in the create client below as:
`url: url || process.env.LOSANT_API_URL || 'https://api.losant.com'`